app/scripts/modules/google/src/loadBalancer/loadBalancer.setTransformer.ts

```diff
@@ -15,10 +15,12 @@ export class GceLoadBalancerSetTransformer {
         name: loadBalancer.name,
         certificate: loadBalancer.certificate,
         ipAddress: loadBalancer.ipAddress,
+        subnet: loadBalancer.subnet,
       };
     });
 
     normalized.name = normalized.urlMapName;
+    delete normalized.subnet;
```

Comment:
What is the purpose of this?

Reply:
Without `delete`, the value is shown at here. (basic section of lb details)
https://github.com/spinnaker/deck/blob/master/app/scripts/modules/google/src/loadBalancer/details/loadBalancerDetails.html#L83

But actually subnet is value of each listeners for Internal HTTP LB.